src/common/config.rs.bk

```diff
@@ -0,0 +1,233 @@
+// Copyright 2017 MaidSafe.net limited.
```

Comment:
either configure gitignore or better, rustfmt with write-mode as overwrite else it might be creating these backup files by default